### PR TITLE
Updated to not use extras 'range' function

### PR DESCRIPTION
**Pull request by shipsaw**
Was unable to build because of dependency on "range", created a PR similar to https://github.com/nektro/zig-extras/commit/1da74fba8c47fde0af1459afa152742821590100

### CODE REVIEW

File: time.zig [hunk 1]
```diff
@@ -187,7 +186,7 @@ pub const DateTime = struct {
 
     pub fn dayOfThisYear(self: Self) u16 {
         var ret: u16 = 0;
-        for (range(self.months), 0..) |_, item| {
+        for (0..self.months, 0..) |_, item| {
```
Comment (nektro) [inline time.zig:189]:
```suggestion
        for (0..self.months) |item| {
```

Comment (shipsaw) [reply time.zig:189]: Yep, that second part was redundant.

Comment (shipsaw) [reply time.zig]: Updated

**Comment by nektro** [comment]
Thanks!

**Pull request merged**